### DISCUSSION TITLE
PR merge button doesn't save preferred option

### DISCUSSION THREAD

**Discussion by gigili** [community/community#6807] [Mobile]
I use the android app for my github work while on the move and when working with PRs I use the squash and merge option when merging PRs. 

And even tho that is the default option for the repo whenever I have a pr I need to click on the little gear icon next to the merge button, reselect the squash and merge option (it already has a green tick next to it ) and than return back to pr for the button to show as squash and merge instead of just merge. 

Not sure if this is a bug in the app state or will it still perform the squash and merge option even tho it shows only merge label on it.

**Comment by brianlovin** [comment]
Hey @gigili I was able to reproduce this, thanks so much for the heads up. I've passed this along to our engineering team, and we'll work on getting a fix up as soon as possible.
